report/4.Design/Performance.tex

```diff
@@ -40,8 +49,10 @@ \subsection{YOLO}
 However, it was decided that making modifications to YOLO would likely be too tedious, while risking that the modifications would still not make it fast enough to run on the Raspberry Pi.
 
 In conclusion, it was determined that the YOLO algorithm would not be feasible for this project.
-\todo{maybe move above to actual conclusion subsection}
 
-\subsection{Conclusion}
-\todo{ ADD PERFORMANCE CONCLUSION WHICH ALGORITHMS WAS CHOSEN BASED ON THESE PERFORMANCE TESTS }
-\todo{ MAYBE CONCLUSION WILL BE NO ML ALGORITHM IS FAST ENOUGH TO USE THEREFORE THE OBJECT FILL ALGORITHM WILL HAVE TO BE USED }
+\subsection{Other machine learning algorithms}
+Since no machine learning algorithms claim to be drastically faster than YOLO, it should be safe to assume that no current machine learning object localization algorithm will be fast enough to run on a Raspberry Pi.
```

Comment:
```suggestion
Since no machine learning algorithms claim to be drastically faster than YOLO, it should be safe to assume that no current machine learning object localization algorithm will be quick enough to run on a Raspberry Pi.
```